internal/fnapi/tenants.go

```diff
@@ -67,3 +72,24 @@ func ExchangeTenantToken(ctx context.Context, token string, scopes []string) (Ex
 
 	return res, nil
 }
+
+func FetchTenantToken(ctx context.Context) (*auth.Token, error) {
+	return tasks.Return(ctx, tasks.Action("nscloud.fetch-tenant-token"), func(ctx context.Context) (*auth.Token, error) {
```

Comment:
"nscloud.fetch-tenant-token" -> "api.fetch-tenant-token"?

Reply:
I opted for "tenants.fetch-tenant-token", as other actions in the same package don't use `api.*` prefix, but rather something more specific (e.g. `dns.*` for naming actions)